docker-compose.yml

```diff
@@ -2,8 +2,14 @@ version: '3.7'
 
 services:
 
-  web:
-    build: docker/
+  php7:
+    container_name: web
```

Comment:
Container names are supposed to be namespaces per project. How else are you going to work on multiple projects at the same time? If you have to hardcode it, at least use `magento2_web`.